### ISSUE TITLE
Score icon doesn't have enough contrast 🧐

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#272]
Color of the icon doesn't have enough contrast. It's hard to read. You can see the [Contrast](https://usecontrast.com/) app showing that there's not enough contrast.

<img width="1680" alt="contrast" src="https://user-images.githubusercontent.com/81969/43592993-c26163ae-9644-11e8-8c5c-37c93dc0971b.png">

**Comment by MariagraziaAlastra** [comment]
This should be fixed by the high contrast icons in #279

**Comment by Zaidtech** [comment]
let me take this issue
